requirements.txt

```diff
@@ -1 +1,2 @@
 PyYAML
+websocket-client
```

Comment:
Could we pin this version?

Reply:
Yeah, definitely. We plan to do that in a separate PR where we pin all the versions (of dev dependencies too). See https://github.com/tinvaan/operator/pull/1

Reply:
@jnsgruk fyi, requirements are pinned to versions in https://github.com/canonical/operator/pull/894